src/pretalx/common/templates/common/auth.html

```diff
@@ -28,11 +28,16 @@ <h4 class="text-center">{% translate "I already have an account" %}</h4>
                 {% translate "Log in" %}
             </button>
         {% endif %}
-        {% socialapp_exists 'eventyay' as eventyay_exists %}
+        {% if request.event and request.event.organiser and request.event.organiser.slug %}
+            {% socialapp_exists request.event.organiser.slug as eventyay_exists %}
+        {% else %}
+            {% with False as eventyay_exists %}
+            {% endwith %}
+        {% endif %}
         {% if eventyay_exists %}
```

Comment:
Does that work? The `with ... as eventyay_exists` is closed by the next line `endwith`, so my guess is that in this case the `eventyay_exists` is not defined in this case?

Reply:
Hi @norbusan, I have moved the eventyay_exists into if condition, so won't need {% with False as eventyay_exists %}